test/widgets/compose_box_test.dart

```diff
@@ -55,13 +56,15 @@ void main() {
 
     final controllerKey = GlobalKey<ComposeBoxController>();
     await tester.pumpWidget(TestZulipApp(accountId: eg.selfAccount.id,
-      child: Column(
-        // This positions the compose box at the bottom of the screen,
-        // simulating the layout of the message list page.
-        children: [
-          const Expanded(child: SizedBox.expand()),
-          ComposeBox(controllerKey: controllerKey, narrow: narrow),
-        ])));
+      child: MediaQuery(
+        data: MediaQueryData(textScaler: textScaler ?? TextScaler.noScaling),
```

Comment:
If we use the global `textScaleFactorTestValue` (see some other tests that use this), we can skip the `MediaQuery` widget, making the test's widget tree more representative.